src/logic/asks_channel.ts

```diff
@@ -255,21 +255,21 @@ const getPermalinkBlocks = async function (
         const daysDifference = Math.round(
           (dateToday.getTime() - messageDate.getTime()) / (1000 * 3600 * 24)
         );
-        const daysMessage =
-          daysDifference == 0
-            ? " (earlier today)"
-            : daysDifference == 1
-            ? " (1 day ago)"
-            : ` (${daysDifference} days ago)`;
-        block.push(
-          createSectionBlock(
-            `<${permalink}|Link to message> from ${
-              message.user
-                ? await getUserDisplayName(message.user)
-                : message.username
-            } at ${messageDate.toLocaleDateString()}${daysMessage}`
-          )
-        );
+        const daysMessage = daysDifference === 0 ? " (earlier today)" : daysDifference === 1 ? " (1 day ago)" : ` (${daysDifference} days ago)`;
+        const userProfile = await getUserProfile(message.user);
+
+        // TODO: Remove the field id's / extract it to a parameter
+        // console.log(`Division ${userProfile?.fields?.Xf01L3FQBAJD?.value}`);
+        // console.log(`Department ${userProfile?.fields?.Xf01KA3W34UX?.value}`);
+        // console.log(`Team ${userProfile?.fields?.Xf01RW57PW6L?.value}`);
+
+        // Limit only 2 words for the team name
```

Comment:
This is to prevent abuse of the team field.
3 words should be enough to cover most team names.